conf/default.toml

```diff
@@ -54,6 +54,9 @@ background_color = "#fdfbff"
 # title_font = "'My Custom Title Font', fallback-font-title"
 # fonts_stylesheet_url = "https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&family=Permanent+Marker&display=swap"
 
+# This optional parameter allow you to change the default class on progress bar, by default is font-bold
+# progress_bar_text_class = 'font-normal'
```

Comment:
I forgot this point but we don't want to introduce this kind of config. You can just get rid of it.
Thanks

Reply:
I need remove the code? did we change the default font by font-normal and let it to font-bold?

Reply:
I think you can remove this change and enforce the font weight of the progress bar from the parent component.

One solution is to use CSS variables in the progress bar component, such as:
```css
.label {
  font-weight: var(--progress-bar-font-weight, 'bold');
}
```

And then from the parent component:
```css
:host {
  --progress-bar-font-weight: 'normal';
}

See a concrete example with this component: https://github.com/geonetwork/geonetwork-ui/blob/main/libs/ui/inputs/src/lib/star-toggle/star-toggle.component.css

Reply:
No I think you should find a solution like the one I suggested